api/spec/controllers/mno_enterprise/deletion_requests_controller_spec.rb

```diff
@@ -58,14 +58,13 @@ def main_app
 
       context 'when the request is pending' do
         it 'freezes the account' do
-          expect(controller.current_user).to receive(:current_deletion_request).and_return(deletion_req)
-          expect(deletion_req).to receive(:freeze_account!)
+          expect_any_instance_of(MnoEnterprise::DeletionRequest).to receive(:freeze_account!)
```

Comment:
but it won't be the same object, as the one passed is a one created from the parsing of the api result...